_pages/security.html

```diff
@@ -35,23 +35,21 @@ <h3>Reporting a Vulnerability</h3>
-          <li>On the embargo date, the Rails Security mailing list is sent a copy of the announcement. The changes are pushed to the public repository and new gems released to <a href="https://rubygems.org/">RubyGems</a>. At least 6 hours after the mailing list is notified, a copy of the advisory will be published on <a href="https://rubyonrails.org/blog/">the Rails weblog</a>.</li>
+          <li>On the embargo date, the Rails Security mailing list is sent a copy of the announcement. The changes are pushed to the public repository and new gems released to <a href="https://rubygems.org/">RubyGems</a>. An annoucement of the releases and a link to the advisory will be published on <a href="https://rubyonrails.org/blog/">the Rails weblog</a>.</li>
         </ol>
-        <p>Typically the embargo date will be set 72 hours from the time <a href="https://oss-security.openwall.org/wiki/mailing-lists/distros">distros@openwall</a> is first notified, however this may vary depending on the severity of the bug or difficulty in applying a fix.</p>
         <p>This process can take some time, especially when coordination is required with maintainers of other projects. Every effort will be made to handle the bug in as timely a manner as possible, however it’s important that we follow the release process above to ensure that the disclosure is handled in a consistent manner.</p>
         <hr class="divider" />
         <h3>Receiving Disclosures</h3>
-        <p>The best way to receive all the security announcements is to subscribe to the <a href="https://groups.google.com/d/forum/rubyonrails-security">Rails Security mailing list</a>. The mailing list is very low traffic, and it receives the public notifications the moment the embargo is lifted. If you produce packages of Ruby on Rails and require prior notification of vulnerabilities, you should be subscribed to <a href="https://oss-security.openwall.org/wiki/mailing-lists/distros">distros@openwall</a>.</p>
-        <p>No one outside the core team, the initial reporter or <a href="https://oss-security.openwall.org/wiki/mailing-lists/distros">distros@openwall</a> will be notified prior to the lifting of the embargo. We regret that we cannot make exceptions to this policy for high traffic or important sites, as any disclosure beyond the minimum required to coordinate a fix could cause an early leak of the vulnerability.</p>
+        <p>The best way to receive all the security announcements is to subscribe to the <a href="https://discuss.rubyonrails.org/c/security-announcements/9">Rails Security Announements section of the forum</a>. This section is very low traffic, and it receives the public notifications the moment the embargo is lifted. If you produce packages of Ruby on Rails and require prior notification of vulnerabilities, you should be subscribed to <a href="https://oss-security.openwall.org/wiki/mailing-lists/distros">distros@openwall</a>.</p>
```

Comment:
Delete `If you produce packages of Ruby on Rails and require prior notification of vulnerabilities, you should be subscribed to <a href="https://oss-security.openwall.org/wiki/mailing-lists/distros">distros@openwall</a>`